studio/data/entity-types/entity-type-query.ts

```diff
@@ -0,0 +1,109 @@
+import { UseQueryOptions } from '@tanstack/react-query'
+import { useCallback } from 'react'
+import { ExecuteSqlData, useExecuteSqlPrefetch, useExecuteSqlQuery } from '../sql/execute-sql-query'
+
+type EntityTypeArgs = {
+  id?: number
+}
+
+export const entityTypeSqlQuery = ({ id }: EntityTypeArgs) => {
+  const sql = /* SQL */ `
+    select
```

Comment:
quick question - this query fetches the singular "entity" based on the `id`
and `entity-types-infinite-query` fetches a list of "entities" right?